models/vrp.rb

```diff
@@ -247,19 +240,18 @@ def self.convert_position_relations(hash)
     end
 
     def self.deduce_solver_parameter(hash)
-      if hash[:configuration] && hash[:configuration][:resolution] && hash[:configuration][:resolution][:solver_parameter]
-        if hash[:configuration][:resolution][:solver_parameter] == -1
-          hash[:configuration][:resolution][:solver] = false
-        else
+      resolution = hash[:configuration] && hash[:configuration][:resolution]
+      if resolution[:solver_parameter].to_i == -1
+        resolution[:solver] = false
+        resolution[:solver_parameter] = nil
+      elsif resolution && resolution[:solver_parameter]
```

Comment:
in this case we should do the following and remove the deprecated parameter from the hash and set the correct parameter.
```ruby
resolution[:solver_parameter] = nil 
resolution[:solver] = true 
```

I know the default value for `solver` is true but we should do this assigment in purpose because solver= true is not because of the default value but  because of deprecated solver_parameter replacement.